exercises/concept/meltdown-mitigation/.meta/exemplar.py

```diff
@@ -67,14 +60,11 @@ def fail_safe(temperature, neutrons_produced_per_second, threshold):
     - `temperature * neutron per second` is not in the above-stated ranges ==  'DANGER'
     """
     output = temperature * neutrons_produced_per_second
-    operational_percentage = int((output / threshold) * 100)
-    safety_range = 'UNKNOWN'
+    operational_percentage = int(output / threshold) * 100
```

Comment:
This is causing test failure, and needs to be backed out for the time being.

```suggestion
    operational_percentage = int(output / threshold * 100)
```